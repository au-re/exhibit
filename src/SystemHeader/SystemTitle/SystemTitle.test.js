import { SystemTitle } from "../../lib";
import React from "react";
import { shallow } from "enzyme";

it("renders without parameters", () => {
  expect(shallow(<SystemTitle />)).toMatchSnapshot();
});

it("renders with an img", () => {
  expect(shallow(<SystemTitle src="/test.jpg" />)).toMatchSnapshot();
});

it("renders with a link", () => {
  expect(shallow(<SystemTitle href="/" />)).toMatchSnapshot();
});

it("renders with a title", () => {
  expect(shallow(<SystemTitle title="hello world" />)).toMatchSnapshot();
});
