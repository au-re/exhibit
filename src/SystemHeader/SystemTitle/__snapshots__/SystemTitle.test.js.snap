// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders with a link 1`] = `
<a
  className="ReactExhibit__SystemTitle"
  href="/"
>
  <span
    className="SystemTitle__Description"
  />
</a>
`;

exports[`renders with a title 1`] = `
<a
  className="ReactExhibit__SystemTitle"
>
  <span
    className="SystemTitle__Description"
  >
    hello world
  </span>
</a>
`;

exports[`renders with an img 1`] = `
<a
  className="ReactExhibit__SystemTitle"
>
  <img
    alt=""
    className="SystemTitle__Image"
    src="/test.jpg"
  />
  <span
    className="SystemTitle__Description"
  />
</a>
`;

exports[`renders without parameters 1`] = `
<a
  className="ReactExhibit__SystemTitle"
>
  <span
    className="SystemTitle__Description"
  />
</a>
`;
